doc/source/block-design.md

```diff
@@ -1,6 +1,6 @@
 # 📐 Block Design
 
-## Random block design
+Random block design
```

Comment:
I think we need this as a second level heading ?

Reply:
ok I will change all the heading again